<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionStorage</class>
 <widget class="QWidget" name="OptionStorage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QChartView" name="chartView">
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_space">
     <property name="topMargin">
      <number>8</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>PRPDlog</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_Asset">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_Others">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>Asset</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>WaveForm</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>DataLog</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_Waveform">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_17">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>其他文件</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_data">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_PRPD">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_fresh">
       <property name="text">
        <string>刷新数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_del_all">
       <property name="text">
        <string>删除数据</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
